### PR TITLE
Bruceper work

### PR DESCRIPTION
**Pull request by BrucePerlerMS**
This checklist is used to make sure that common issues in a pull request are addressed. This will expedite the process of getting your pull request merged and avoid extra work on your part to fix issues discovered during the review process. 

### PR information
- [ ] The title of the PR is clear and informative.
- [ ] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For information on cleaning up the commits in your pull request, [see this page](https://github.com/Azure/azure-powershell/blob/dev/documentation/cleaning-up-commits.md).
- [ ] Except for special cases involving multiple contributors, the PR is started from a fork of the main repository, not a branch.
- [ ] If applicable, the PR references the bug/issue that it fixes.
- [ ] Swagger files are correctly named (e.g. the `api-version` in the path should match the `api-version` in the spec).

### Quality of Swagger
- [ ] I have read the [contribution guidelines](https://github.com/Azure/azure-rest-api-specs/blob/master/.github/CONTRIBUTING.md).
- [ ] My spec meets the review criteria:
  - [ ] The spec conforms to the [Swagger 2.0 specification](https://github.com/OAI/OpenAPI-Specification/blob/master/versions/2.0.md).
  - [ ] The spec follows the guidelines described in the [Swagger checklist](../documentation/swagger-checklist.md) document.
  - [ ] [Validation tools](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md#validation-tools-for-swagger-checklist) were run on swagger spec(s) and have all been fixed in this PR.

### CODE REVIEW

**Comment by BrucePerlerMS** [comment]
comment updates for most/all API descriptions. Please review

File: keyvault/2016-10-01/swagger/keyvault.json [hunk 1]
```diff
@@ -158,7 +161,8 @@
           "Keys"
         ],
         "operationId": "UpdateKey",
-        "description": "The update key operation changes specified attributes of a stored key and can be applied to any key type and key version stored in Azure Key Vault. The cryptographic material of a key itself cannot be changed. In order to perform this operation, the key must already exist in the Key Vault. Authorization: requires the keys/update permission.",
+        "summary": "The update key operation changes specified attributes of a stored key and can be applied to any key type and key version stored in Azure Key Vault.",
+        "description": "The cryptographic material of a key itself cannot be changed. In order to perform this operation, the key must already exist in the Key Vault.",
```
Comment (dsgouda) [inline keyvault/2016-10-01/swagger/keyvault.json]: nit: doesn't really describe the operation itself

File: keyvault/2016-10-01/swagger/keyvault.json [hunk 2]
```diff
@@ -516,7 +527,8 @@
           "Keys"
         ],
         "operationId": "sign",
-        "description": "Creates a signature from a digest using the specified key.",
+        "summary": "Creates a signature from a digest using the specified key.",
+        "description": "SIGN is applicable to asymmetric and symmetric keys stored in Azure Key Vault since this operation uses the private portion of the key.",
```
Comment (dsgouda) [inline keyvault/2016-10-01/swagger/keyvault.json]: nit: You could say "The SIGN operation is applicable..." just like all other operations to maintain consistency and provide better description.

File: keyvault/2016-10-01/swagger/keyvault.json [hunk 3]
```diff
@@ -568,7 +580,8 @@
           "Keys"
         ],
         "operationId": "verify",
-        "description": "Verifies a signature using a specified key.",
+        "summary": "Verifies a signature using a specified key.",
+        "description": "VERIFY is applicable to symmetric keys stored in Azure Key Vault. VERIFY is not strictly necessary for asymmetric keys stored in Azure Key Vault since signature verification can be performed using the public portion of the key but this operation is supported as a convenience for callers that only have a key-reference and not the public portion of the key.",
```
Comment (dsgouda) [inline keyvault/2016-10-01/swagger/keyvault.json]: nit: "VERIFY operation"

File: keyvault/2016-10-01/swagger/keyvault.json [hunk 4]
```diff
@@ -984,7 +1005,8 @@
           "Certificates"
         ],
         "operationId": "GetCertificates",
-        "description": "List certificates in a specified key vault",
+        "summary": "List certificates in a specified key vault",
+        "description": " ",
```
Comment (dsgouda) [inline keyvault/2016-10-01/swagger/keyvault.json]: No description?

File: keyvault/2016-10-01/swagger/keyvault.json [hunk 5]
```diff
@@ -1060,7 +1083,8 @@
           "Certificates"
         ],
         "operationId": "SetCertificateContacts",
-        "description": "Sets the certificate contacts for the specified key vault.",
+        "summary": "Sets the certificate contacts for the specified key vault.",
+        "description": " ",
```
Comment (dsgouda) [inline keyvault/2016-10-01/swagger/keyvault.json]: Empty description

**Review by dsgouda — CHANGES_REQUESTED** [review]
Operation descriptions should be provided and some nit comments

**Comment by BrucePerlerMS** [comment]
@Pooneh Mortazavi - Please have a look at these documentation updates to the swagger file and accept or offer feedback. Thanks, Bruce

File: keyvault/2016-10-01/swagger/keyvault.json (hunk 4 above)
Comment (BrucePerlerMS) [inline keyvault/2016-10-01/swagger/keyvault.json]: I'm in touch with the dev and will be updating this.

File: keyvault/2016-10-01/swagger/keyvault.json (hunk 5 above)
Comment (BrucePerlerMS) [inline keyvault/2016-10-01/swagger/keyvault.json]: I'm in touch with the dev and will be updating this.

Comment (dsgouda) [inline keyvault/2016-10-01/swagger/keyvault.json]: FYI there are multiple occurrences I have pointed out just a couple.

File: keyvault/2016-10-01/swagger/keyvault.json (hunk 1 above)
Comment (BrucePerlerMS) [inline keyvault/2016-10-01/swagger/keyvault.json]: Its an Key update operation that takes the specified attributes as arguments and sets those attrs. What more would you like to see?

File: keyvault/2016-10-01/swagger/keyvault.json (hunk 2 above)
Comment (BrucePerlerMS) [inline keyvault/2016-10-01/swagger/keyvault.json]: will do

File: keyvault/2016-10-01/swagger/keyvault.json (hunk 3 above)
Comment (BrucePerlerMS) [inline keyvault/2016-10-01/swagger/keyvault.json]: will do

File: keyvault/2016-10-01/swagger/keyvault.json (hunk 1 above)
Comment (dsgouda) [inline keyvault/2016-10-01/swagger/keyvault.json]: I probably should have worded it as this comment https://github.com/Azure/azure-rest-api-specs/pull/1114/files/08da994f2d15e01c3d1c9d54c64fe172857dc15d#diff-778ad7ebcca7d8ff16a50b1ef232a622

**Comment by BrucePerlerMS** [comment]
Hi @pomortaz, 
Please have a look at the swagger updates here. I'm also about to sync with some placeholder descriptions for the Certificate operations, which are thin on detail. 
Thanks,
Bruce

File: keyvault/2016-10-01/swagger/keyvault.json (hunk 5 above)
Comment (BrucePerlerMS) [inline keyvault/2016-10-01/swagger/keyvault.json]: I've gone through the entire file and filled in placeholders, per the note below. Working with @pomortaz to get fuller descriptions as the responsibility for this is with the dev team.

**Comment by BrucePerlerMS** [comment]
@pomortaz - when I began editing the swagger file I found most of the description content was in the "description" field on the operation and there was no "summary" field. I've edited the descriptions, created summaries for all and have placeholder descriptions for many of the CERT resource ops due to lack of details. 

The swagger file is to contain only language neutral descriptions. This way the language specific info can be added in a downstream step using overwrites. For instance, for REST, the guidance about using nextLink is contained in the overwrite for ops that return lists / sets.

**Comment by pomortaz** [comment]
@BrucePerlerMS the changes look good. Thanks!
However, please do not check in the changes without filling in the cert descriptions.

**Comment by BrucePerlerMS** [comment]
Hi Pooneh,
The content for the cert descriptions need to come from your team. I'd fill them in if I had the details.
😊 Bruce

Sent from my Windows Phone
________________________________

**Comment by dsgouda** [comment]
@BrucePerlerMS @pomortaz 
Do we have any updates here, please reply soon since the PR has been idle for almost a week now.

**Comment by BrucePerlerMS** [comment]
This is an important PR that is being worked with by the Azure Key Vault team. Please be patient with the process.

Thank you,
Bruce

**Comment by BrucePerlerMS** [comment]
@pomortaz - I've discussed this swagger file with the AKV prod team and we've determined that it should be merged as is. The less-than-complete Cert Resource descriptions will be addressed in a separate process. Please sign-off.

#sign-off

**Review by dsgouda — APPROVED** [review]

**Pull request merged**